GETTING_STARTED.md

```diff
@@ -1,84 +0,0 @@
-# Getting started with Beluga
-
-## Crash course
-
-1. **Clone the repository**. You will need `git`.
-
-   ```bash
-   git clone --recursive git@github.com:Ekumen-OS/beluga.git
-   ```
-
-1. **Build and run the development docker container**. You will need [`docker-compose v2.10+`](https://github.com/docker/compose/tree/v2).
-
-   ```bash
-   (cd beluga && docker/run.sh)
-   ```
-   To rebuild the image before starting the container, use:
-   ```bash
-   (cd beluga && docker/run.sh --build)
-   ```
-   To target an specific ROS distribution, use:
-   ```bash
-   (cd beluga && ROSDISTRO=humble docker/run.sh)
-   ```
-   Supported distributions include `noetic`, `humble`, `iron`, and `rolling`.
-
-1. **Build and test the project** (inside development container).
-
-    ```bash
-    cd /ws
-    colcon build --symlink-install
-    colcon test
-    ```
-
-    You may also use `catkin_make_isolated` and `catkin-tools` in ROS 1 distributions.
-
-1. **Run an example application using a ROS bag** (inside development container).
-
-    For ROS 2 distributions, run:
-    ```bash
-    cd /ws
-    source install/setup.bash
-    ros2 launch beluga_example perfect_odometry.launch.xml
-    ```
-
-    For ROS 1 distributions, run:
-    ```bash
-    cd /ws
-    source devel*/setup.bash
-    roslaunch beluga_example perfect_odometry.launch
-    ```
-
-1. **Run an example application using a simulation and teleop controls** (inside development container).
-
-    For ROS 2 distributions, in two separate terminals run:
-    ```bash
-    cd /ws
-    source install/setup.bash
-    ros2 launch beluga_example simulation.launch.xml
-    ```
-    ```bash
-    ros2 run teleop_twist_keyboard teleop_twist_keyboard
-    ```
-
-    For ROS 1 distributions, in two separate terminals run:
-    ```bash
-    cd /ws
-    source devel*/setup.bash
-    roslaunch beluga_example simulation.launch
-    ```
-    ```bash
-    rosrun teleop_twist_keyboard teleop_twist_keyboard
```

Comment:
If the steps in this file are followed, when tried running with **ROS Noetic**, there's only a **.py** file to run teleop_twist_keyboard.
Then, it only works if the command is : `rosrun teleop_twist_keyboard teleop_twist_keyboard.py`.

Reply:
Thanks! Fixed in https://github.com/Ekumen-OS/beluga/pull/346/commits/3e9961897fcd9f2f23db5253b2d018be630b0f11